inductiva/fluids/simulators/xbeach.py

```diff
@@ -22,7 +22,8 @@ def run(
         output_dir: Optional[types.Path] = None,
         resource_pool_id: Optional[UUID] = None,
         n_cores: int = 1,
-    ) -> pathlib.Path:
+        run_async: bool = False,
+    ) -> Union[pathlib.Path, Task]:
```

Comment:
I don't really like the way that we can have different outputs depending on the run_async argument. But I think it is something we should discuss in a design meeting.